README.md

```diff
@@ -48,20 +44,36 @@ by running it with the `--help` flag or reading the [rustc dev guide][rustcguide
 [gettingstarted]: https://rustc-dev-guide.rust-lang.org/getting-started.html
 [rustcguidebuild]: https://rustc-dev-guide.rust-lang.org/building/how-to-build-and-run.html
 
-### Building on a Unix-like system
-1. Make sure you have installed the dependencies:
+### Dependencies
+
+Make sure you have installed the dependencies:
 
-   * `g++` 5.1 or later or `clang++` 3.5 or later
```

Comment:
I think we probably have C/C++ dependencies somewhere in the compiler tree? At minimum, these are needed for any Rust compilation as we typically invoke the linker through cc, right? (Not sure about Windows, but I believe this is the case on Ubuntu at least).

Reply:
We need at least cc, but not a full C++ compiler. I can say that explicitly if you like, but I think most people assume they need at least a C compiler for most things.

Reply:
Hm. I guess I would probably not call that an implicit assumption we can make; I think we should say it. Folks coming from languages like Node or Python might not realize that they need extra compilers to work on std, for example.

(And long-term I think we should try to remove this system dependency, e.g., by including clang in our artifacts. But that's a very separate conversation).